src/UIKit/UISearchController.cs

```diff
@@ -17,6 +18,8 @@ public __Xamarin_UISearchResultsUpdating (Action<UISearchController> cback)
 				this.cback = cback;
 				IsDirectBinding = false;
 			}
+
+			[Preserve (Conditional = true)] // called back from native, no direct managed reference (except on the type itself)
```

Comment:
Shouldn't overriding an exported method be enough to make the linker preserve it?

Reply:
You're right - the `override` should have been enough to automagically preserve it (and it would explain why it, _likely_, worked before). It's possible that logic got broken and, in most cases (user code) the override would not be linked (so this would hide the issue)